### PR TITLE
GRIM: Add hebrew fan translation detection entry

### PR DESCRIPTION
**Pull request by BLooperZ**
Following #2880
This adds detection entry for hebrew fan translation

Also, fixes off-by-1 error for the string reversal

Thanks

### CODE REVIEW

File: engines/grim/textobject.cpp [hunk 1]
```diff
@@ -283,7 +283,7 @@ void TextObject::setupText() {
 		const char *start = message.c_str();
 		// Reverse the line for the Hebrew translation
 		if (g_grim->getGameLanguage() == Common::HE_ISR) {
-			for (const char *ch = start + nextLinePos - 1; ch > start; --ch)
+			for (const char *ch = start + nextLinePos - 1; ch >= start; --ch)
```
Comment (orgads) [inline engines/grim/textobject.cpp:286]: Can you please split this fix to a separate commit?

Comment (BLooperZ) [reply engines/grim/textobject.cpp:286]: done👍

File: engines/grim/detection.cpp [hunk 2]
```diff
@@ -153,6 +153,21 @@ static const GrimGameDescription gameDescriptions[] = {
 		},
 		GType_GRIM
 	},
+	{
+		// Grim Fandango Hebrew Fan translation (patched)
+		// As English version uses VOX0001.LAB, this needs to be listed first
```
Comment (bluegr) [inline engines/grim/detection.cpp:158]: I don't understand this comment. This entry is not listed first

Comment (BLooperZ) [reply engines/grim/detection.cpp]: At first it was, then I saw it doesn't have to be first and forgot to remove the comment, now it's removed

**Review by orgads — APPROVED** [review]

**Comment by BLooperZ** [comment]
Replaced the line reversal fpr visual hebrew with unicode-bidi

**Comment by BLooperZ** [comment]
As requested by @orgads 
the changes regarding BiDi support have moved to separate PR: #2917

File: engines/grim/detection.cpp [hunk 3]
```diff
@@ -153,6 +153,20 @@ static const GrimGameDescription gameDescriptions[] = {
 		},
 		GType_GRIM
 	},
+	{
+		// Grim Fandango Hebrew Fan translation (patched)
+		{
+			"grim",
+			"",
+			AD_ENTRY2s("LOCAL.LAB", "d22648d6787c2f8f0a789ee3ed0c08f7", 608,
```
Comment (orgads) [inline engines/grim/detection.cpp:161]: What's LOCAL.LAB? I don't have it. Can't find it anywhere in the [installation guide](https://www.facebook.com/groups/200491360554968/permalink/356883714915731) either.

**Comment by orgads** [comment]
Thanks.
